main.go

```diff
@@ -289,6 +294,16 @@ func addDependencies(_ context.Context, mgr ctrl.Manager, cfg config.Config, v v
 	return nil
 }
 
-func minTlsDefault(cfg *tls.Config) {
-	cfg.MinVersion = defaultMinTLSVersion
+func tlsConfigSetting(cfg *tls.Config, tlsOpt tlsConfig) {
```

Comment:
The comment would be appreciated why this functionality is here (not only what it does)

Reply:
OK i will add in!! hehe

Reply:
@pavolloffay added now